prow/crier/reporters/github/reporter.go

```diff
@@ -164,19 +170,19 @@ func pjsToReport(ctx context.Context, log *logrus.Entry, lister ctrlruntimeclien
 	}
 	var pjs v1.ProwJobList
 	if err := lister.List(ctx, &pjs, ctrlruntimeclient.MatchingLabels(selector)); err != nil {
-		return nil, fmt.Errorf("Cannot list prowjob with selector %v", selector)
+		return nil, false, fmt.Errorf("Cannot list prowjob with selector %v", selector)
 	}
 
 	latestBatch := make(map[string]v1.ProwJob)
 	for _, pjob := range pjs.Items {
 		if !pjob.Complete() { // Any job still running should prevent from comments
-			return nil, nil
```

Comment:
there is no job returned when there is any job still running with the original implementation, and the report would be skipped https://github.com/kubernetes/test-infra/blob/8fd98f95dc8f7cd34bfccc19277da88a78c598f7/prow/github/report/report.go#L167

Reply:
Ahhh, I think you are right. I didn't notice https://github.com/kubernetes/test-infra/blob/d6853a939d8fc13a2a2491417615657ec1b4386f/prow/github/report/report.go#L166-L168

Reply:
Speculation, but trigger cancels all jobs before it creates new ones so I suppose it is possible to observe a state where there are no jobs: https://github.com/kubernetes/test-infra/blob/b74506333229b4cecfddccded988e80f5742e4b3/prow/plugins/trigger/pull-request.go#L119-L122

What is surprising to me though is that triggers sets it to aborted but not complete and then plank has to complete it. I wouldn't really expect plank to regularly be faster at completing jobs than trigger at creating even a single new one?

If we are convinced the above explanation is correct, we could inverse that order in trigger? It will be slightly complicated though, right now we just abort everything for the PR, which we can't do anymore if we inverse the order.

Reply:
Thanks for the info @alvaroaleman. With that in mind I took a new approach here.